x/conflict/README.md

```diff
@@ -0,0 +1,124 @@
+For the conflict resolution there needs to be a majority met of votes for Provider A, Provider B or None of them. 
+If a majority was not met, conflict reward pool is given to the consumer that reported the conflict.
+Once a majority is met providers that voted to the wrong side of the conflict are slashed and frozen, the slashed amount is added to the conflict reward pool.
+Now the reward pool is distributed between the comsumer and the providers that voted for the correct provider.
+
+## Parameters
+
+The conflict module contains the following parameters:
+
+| Key                                    | Type                    | Default Value    |
+| -------------------------------------- | ----------------------- | -----------------|
+| MajorityPercent                        | math.LegacyDec          | 0.95              |
+| VoteStartSpan                              | uint64          | 3             |
+| VotePeriod                       | uint64          | 2                |
+| Rewards                        | rewards                  | N/A                |
+
+`MajorityPercent` majority needed to conclude who is the winner of the conclict
+`VoteStartSpan` is the amount of epochs in the past that a consumer can send conflict detection message for
+`VotePeriod` is the amount of epochs for each voting state (commit/reveal)
+`Rewards` defines how the reward pool tokens will be distributed between all participants
+
+```go
+type Rewards struct {
+	WinnerRewardPercent github_com_cosmos_cosmos_sdk_types.Dec // the winnig provider of the conflict portion
+	ClientRewardPercent github_com_cosmos_cosmos_sdk_types.Dec // the reporter of the conflict portion
+	VotersRewardPercent github_com_cosmos_cosmos_sdk_types.Dec // the jury portion
+}
+```
+## Queries
+
+The Conflict module supports the following queries:
+
+| Query             | Arguments         | What it does                                  |
+| ----------        | ---------------   | ----------------------------------------------|
+| `params`          | none              | show the params of the module                 |
+| `consumer-conflicts` | consumer (string)              | shows all conflicts reported and active by the consumer         |
+| `list-conflict-vote` | none           | shows all active conflicts                |
+| `show-conflict-vote`       | voteID (string)           | shows a specific active conflict                             |
+
+## Transactions
+
+The Conflict module transactions are not ment to be sent by the CLI and are used by the consumer and providers.
+for more information look [here](../../proto/lavanet/lava/conflict/tx.proto)
+
+### Events
+
+The plans module has the following events:
+| Event             | When it happens       |
+| ----------        | --------------- |
+| `response_conflict_detection`        | a new conflict has opened (this event includes all of the jury providers) and is entering the commit stage  |
```

Comment:
-> A new conflict has been opened, which involves all of the jury providers. It is now entering the commit stage

Reply:
done